test-multiple-bootstrap-config.yaml

```diff
@@ -0,0 +1,11 @@
+#
```

Comment:
I guess this testing config should be deleted

Reply:
I guess I will remove it when the MR is ready to merge. I might still need it for testing later.